src/main/java/persistence/sql/ddl/dialect/H2Dialect.java

```diff
@@ -0,0 +1,27 @@
+package persistence.sql.ddl.dialect;
+
+import persistence.sql.ddl.EntityField;
+import persistence.sql.ddl.exception.NotSupportException;
+
+import java.sql.Types;
+
+public class H2Dialect implements Dialect{
```

Comment:
```suggestion
public class H2Dialect implements Dialect {
```
Dialect 👍 

결국 id라고 하여도 일반 column과 타입이 동일하게 사용될텐데 id Field와 일반 field를 구분하여 메소드를 정의해주신 이유가 있을까요?

Reply:
음.. 처음에 Person.class 만 봤을때는 int, long만 쓰겠지. 해서 좀 더 타이트하게 한건데요~ 제거해도 될것 같네요!